### PR TITLE
Windows volume fix.

### PR DESCRIPTION
**Pull request by intesar**
When deploying containers to windows with volumes - the docker-java API throws error when parsing windows paths like c:\dir:c:\dir.

This fixes windows paths like.
1. C:\dir:C:\dir
2. C:\dir:C:\dir:RO

---
This change is [Reviewable](https://reviewable.io/reviews/docker-java/docker-java/919)

### CODE REVIEW

**Comment by intesar** [comment]
The code was tested on 3.0.4. If you're merging into master you may have to resolve merge conflicts.

**Comment by KostyaSha** [comment]
Changed targe branch, but there is conflict, could you rebase?

**Comment by intesar** [comment]
Here is the clean pull with rebased to 3.0.x.
https://github.com/docker-java/docker-java/pull/920

File: src/main/java/com/github/dockerjava/api/model/Bind.java [hunk 1]
```diff
@@ -52,44 +53,98 @@ public SELContext getSecMode() {
...
-                    if (p.length() == 2) {
-                        accessMode = AccessMode.valueOf(p.toLowerCase());
-                    } else {
-                        seMode = SELContext.fromString(p);
-                    }
-                }
+            String[] parts = null;
 
-                return new Bind(parts[0], new Volume(parts[1]), accessMode, seMode);
+            if (SystemUtils.IS_OS_WINDOWS) {
```
Comment (KostyaSha) [inline src/main/java/com/github/dockerjava/api/model/Bind.java]: path creation shouldn't depend on platform where jvm is running

**Comment by vindiagram** [comment]
Was this issue ever resolved???

**Pull request closed without merging**